client/src/components/modal/component.tsx

```diff
@@ -0,0 +1,112 @@
+import { useRef } from 'react';
```

Comment:
Modal from [Vizzuality/front-end-scaffold](https://github.com/Vizzuality/front-end-scaffold/tree/main/components/modal), with tweaks to fit this project's design.